### PR TITLE
Gh5850 Added netamount and grossamount columns

### PR DESCRIPTION
**Pull request by nelucristian**
https://github.com/metasfresh/metasfresh/issues/5850

### CODE REVIEW

File: de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5537330_sys_gh5850_add_netamount_grossamount_columns.sql [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+;
+
+-- 2019-11-28T08:04:31.687Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+/* DDL */  select update_Column_Translation_From_AD_Element(577397) 
+;
+
+-- 2019-11-29T10:03:12.930Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Field (AD_Client_ID,AD_Column_ID,AD_Field_ID,AD_Org_ID,AD_Tab_ID,ColumnDisplayLength,Created,CreatedBy,DisplayLength,EntityType,IncludedTabHeight,IsActive,IsDisplayed,IsDisplayedGrid,IsEncrypted,IsFieldOnly,IsHeading,IsReadOnly,IsSameLine,Name,SeqNo,SeqNoGrid,SortNo,SpanX,SpanY,Updated,UpdatedBy) VALUES (0,569682,592818,0,540737,0,TO_TIMESTAMP('2019-11-29 12:03:12','YYYY-MM-DD HH24:MI:SS'),100,0,'D',0,'Y','Y','Y','N','N','N','N','N','Nettwert',90,90,0,1,1,TO_TIMESTAMP('2019-11-29 12:03:12','YYYY-MM-DD HH24:MI:SS'),100)
```
Comment (metas-ts) [inline de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5537330_sys_gh5850_add_netamount_grossamount_columns.sql]:
```suggestion
INSERT INTO AD_Field (AD_Client_ID,AD_Column_ID,AD_Field_ID,AD_Org_ID,AD_Tab_ID,ColumnDisplayLength,Created,CreatedBy,DisplayLength,EntityType,IncludedTabHeight,IsActive,IsDisplayed,IsDisplayedGrid,IsEncrypted,IsFieldOnly,IsHeading,IsReadOnly,IsSameLine,Name,SeqNo,SeqNoGrid,SortNo,SpanX,SpanY,Updated,UpdatedBy) VALUES (0,569682,592818,0,540737,0,TO_TIMESTAMP('2019-11-29 12:03:12','YYYY-MM-DD HH24:MI:SS'),100,0,'D',0,'Y','Y','Y','N','N','N','N','N','Nettowert',90,90,0,1,1,TO_TIMESTAMP('2019-11-29 12:03:12','YYYY-MM-DD HH24:MI:SS'),100)
```

File: de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5537330_sys_gh5850_add_netamount_grossamount_columns.sql [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+
+
+
+
+
+
+-- 2019-11-29T10:38:54.062Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_UI_Element (AD_Client_ID,AD_Field_ID,AD_Org_ID,AD_Tab_ID,AD_UI_Element_ID,AD_UI_ElementGroup_ID,AD_UI_ElementType,Created,CreatedBy,IsActive,IsAdvancedField,IsAllowFiltering,IsDisplayed,IsDisplayed_SideList,IsDisplayedGrid,IsMultiLine,MultiLine_LinesCount,Name,SeqNo,SeqNo_SideList,SeqNoGrid,Updated,UpdatedBy) VALUES (0,592818,0,540737,564374,1000030,'F',TO_TIMESTAMP('2019-11-29 12:38:53','YYYY-MM-DD HH24:MI:SS'),100,'Y','N','N','Y','N','N','N',0,'Nettwert',20,0,0,TO_TIMESTAMP('2019-11-29 12:38:53','YYYY-MM-DD HH24:MI:SS'),100)
```
Comment (metas-ts) [inline de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5537330_sys_gh5850_add_netamount_grossamount_columns.sql]:
```suggestion
INSERT INTO AD_UI_Element (AD_Client_ID,AD_Field_ID,AD_Org_ID,AD_Tab_ID,AD_UI_Element_ID,AD_UI_ElementGroup_ID,AD_UI_ElementType,Created,CreatedBy,IsActive,IsAdvancedField,IsAllowFiltering,IsDisplayed,IsDisplayed_SideList,IsDisplayedGrid,IsMultiLine,MultiLine_LinesCount,Name,SeqNo,SeqNo_SideList,SeqNoGrid,Updated,UpdatedBy) VALUES (0,592818,0,540737,564374,1000030,'F',TO_TIMESTAMP('2019-11-29 12:38:53','YYYY-MM-DD HH24:MI:SS'),100,'Y','N','N','Y','N','N','N',0,'Nettowert',20,0,0,TO_TIMESTAMP('2019-11-29 12:38:53','YYYY-MM-DD HH24:MI:SS'),100)
```

File: de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5537330_sys_gh5850_add_netamount_grossamount_columns.sql (hunk 1 above)
Comment (metas-ts) [inline de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5537330_sys_gh5850_add_netamount_grossamount_columns.sql]: Don't forget then en_US translation.
When Adding new AD_ELements in the metasfresh repository, we generally try to have a proper de_DE, de_CH (almost always identical with de_DE) and en_US translation for.

Comment (nelucristian) [reply de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5537330_sys_gh5850_add_netamount_grossamount_columns.sql]: Done. Please review again

**Pull request merged**